1.6/Defs/Things/Resin.xml

```diff
@@ -112,4 +112,4 @@
     <allowedArchonexusCount>80</allowedArchonexusCount>
   </ThingDef>
 
-</Defs>
\ No newline at end of file
+</Defs>

```

Comment:
POSIX-compliant line ending. It is a "mostly-irrelevant-seldomly-good" idea to have one.

If you do not like it, I will remove it.